packages/arb-token-bridge-ui/src/util/WithdrawalUtils.ts

```diff
@@ -105,3 +106,54 @@ export async function withdrawInitTxEstimateGas({
     }
   }
 }
+
+const SECONDS_IN_MINUTE = 60
+const SECONDS_IN_HOUR = 3600
+const SECONDS_IN_DAY = 86400
+const DEFAULT_CONFIRMATION_TIME = 7 * SECONDS_IN_DAY
+const DEFAULT_FAST_WITHDRAWAL_TIME = SECONDS_IN_DAY
+
+function formatDuration(seconds: number): string {
+  if (seconds < SECONDS_IN_MINUTE) return `${seconds} seconds`
+  if (seconds < SECONDS_IN_HOUR)
+    return `${Math.round(seconds / SECONDS_IN_MINUTE)} minutes`
+  if (seconds < SECONDS_IN_DAY)
+    return `${Math.round(seconds / SECONDS_IN_HOUR)} hours`
+  return `${Math.round(seconds / SECONDS_IN_DAY)} days`
+}
```

Comment:
we normally use dayjs for formatting duration / date / time, check if this function is really needed?